packages/dataviews/src/components/dataviews-bulk-actions/index.tsx

```diff
@@ -90,12 +93,25 @@ export function useSomeItemHasAPossibleBulkAction< Item >(
 	}, [ actions, data ] );
 }
 
+export function useIsMultiselectablePicker< Item >(
```

Comment:
Not at all a real issue but `isMultiselectablePicker` reads a little long to me. It's an internal hook so doesn't matter at all, but it we wanted to tighten it up, we could call it `isMultiselectPicker`.